cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -65,9 +73,13 @@ func NewPasstNetworkConfigurator(ifaces []vmschema.Interface, networks []vmschem
 	if iface.Binding == nil || iface.Binding != nil && iface.Binding.Name != PasstPluginName {
 		return nil, fmt.Errorf("interface %q is not set with Passt network binding plugin", network.Name)
 	}
+	if opts.NetworkHandler == nil {
+		opts.NetworkHandler = &netlink.NetLink{}
```

Comment:
If I understand correctly, the unit tests that do not use a stub/fake netlink, will in practice try to call `netlink` and look for the interface. We do not really want to mix the OS into this, so this is a good reasoning for not making the link getter optional. But it can be solved by making sure the tests always pass in a stub that returns the error.